.github/workflows/ci.yml

```diff
@@ -127,7 +127,7 @@ jobs:
         path: artifacts.tar.gz
 
   linux-arm64:
-    runs-on: linux-arm64
+    runs-on: ${{ github.repository == 'openssl/openssl' && 'linux-arm64' || 'ubuntu-24.04-arm' }}
```

Comment:
why do we want to use either linux-arm64 or ubuntu-24.04-arm?  Is that just to gain the latency advantage of whatever runner is available at the moment?

Reply:
I dont know why, but whenever I push something on my own github account every test run works (rather slow, but it works), except this runner which waits for ever, in the end I have to cancel each "Github CI" workfow, and therefore I get no automatic e-mail when any test run in this workflow fails, because that waits also for the linux-arm64 runner.
But the ubuntu-24-04-arm workflow is available for free, however it is a beta version.  I did not want to interfere with the official workflow, all other people I know seem to have disabled all their personal github workflows on their repository, probably because of these issues.

Reply:
Wouldn't it make more sense to switch over to `ubuntu-24-04-arm` for the project too?

Reply:
I couldn't find any public documentation about `linux-arm64` and what might be the difference to `ubuntu-24-04-arm` therefore I figured that it might offer better availability as it is obviously not available for everyone.